Game/HidingSystem.gd

```diff
@@ -4,12 +4,13 @@ var hiding_spots = []
 var current_hiding_spot
 
 signal print_dialogue
+signal new_hiding_spot(interactable: Interactable)
 
 func _ready():
     sb.add_emitter("print_dialogue", self)
+    sb.add_emitter("new_hiding_spot", self)
```

Comment:
This adds our local signal (defined on line 7) to be picked up by the SignalBus component and passed to any listeners in the global context.